ivy/functional/ivy/activations.py

```diff
@@ -609,3 +609,50 @@ def mish(
     }
     """
     return current_backend(x).mish(x, out=out)
+
+
+@handle_exceptions
+@handle_nestable
+@handle_array_like_without_promotion
+@handle_out_argument
+@to_native_arrays_and_back
+@handle_array_function
+def hardswish(
+    x: Union[ivy.Array, ivy.NativeArray], out: Optional[ivy.Array] = None
+) -> ivy.Array:
```

Comment:
The ` /, *,` specification in the signature should be consistent across all implementations. I suppose that's the reason you are getting the positional vs keyword error.